impress-org/givewp
# Improve loading spinner animation in Legacy form

Resolves #5395 

## Description
This PR is improving the loading spinner animation used in the Legacy form. 
The issue is solved by entering the correct value in CSS definition responsible for the spinner animation. 

## Affects

Legacy form loading spinner.

## Visuals

![deepin-screen-recorder_Select area_20201020183955](https://user-images.githubusercontent.com/4222590/96617300-bc4e2680-1303-11eb-8594-b1b2c5cd29ec.gif)


## Pre-review Checklist




## Testing Instructions

1. Create legacy form and insert it into a new or existing page.
2. Visit that page and fill in all Donation form fields.
3. Click on the "Donate now" button
4. The loading spinner should appear, but this time it should run smoothly
